### ISSUE TITLE
Floating Label does not reset position when input is cleared

### ISSUE THREAD

**Issue by JoeNimble** [GeekyAnts/NativeBase#1079]
Floating label is not resetting to the blurred state when clearing input on both ios and android.

## Versions
native-base: 2.1.5
react: 16.0.0-alpha.12
react-native: 0.45.0

## Example
```
<Item floatingLabel light ref={item => { this.emailItem= item; }}>
  <Label>Email</Label>
  <Input
     getRef={input => { this.emailInput = input; }}
     returnKeyType='next'
     keyboardType='email-address'
     autoCapitalize='none'
     autoCorrect={false}
     onChangeText={email => this.setState({ email })}
     onSubmitEditing={() => this.submit()}
     value={this.state.email}
  />
</Item>
```
```
submit = () => {
  //these do not reset the floating label
  this.setState({email: ''});
  this.emailInput._root.clear();
  this.emailInput._root.blur();
  //this.emailItem._inputRef.clear();   //is undefined
  //this.emailItem._inputRef.blur();   //is undefined
}
```
## Screenshots
Initial empty state:
<img src="https://user-images.githubusercontent.com/10679867/28401170-81cbc6d0-6d5b-11e7-9616-4c2af1a83ba2.png" width=350 />

Entering text:
<img src="https://user-images.githubusercontent.com/10679867/28401072-afd21206-6d5a-11e7-9354-66647586481b.png" width=350 />

After clear:
<img src="https://user-images.githubusercontent.com/10679867/28401077-b8edc29a-6d5a-11e7-9e7d-860b139054c5.png" width=400 />

Expected `clear()` to reset floating label position (in the example, email label should look like password)

**Comment by lukecwilliams** [comment]
Also running into this issue.  Have tried `.clear()` as above, as well as

    this._TextInput._root.setNativeProps({text: ""});

**Comment by SupriyaKalghatgi** [comment]
@JoeNimble @lukecwilliams Did you check this with NativeBase 2.3.3?

**Comment by JoeNimble** [comment]
@SupriyaKalghatgi still an issue for me using 2.3.3

**Comment by harshada-bhanose** [comment]
@SupriyaKalghatgi still facing this issue using 2.3.3

**Comment by Nachox07** [comment]
@SupriyaKalghatgi still facing this issue using 2.3.3

**Comment by jribeiro** [comment]
Hi guys, any update on this? Would you be happy to accept a PR to address it?

**Comment by SupriyaKalghatgi** [comment]
@jribeiro Sure!

**Comment by rizzomichaelg** [comment]
PR #1606

**Comment by rizzomichaelg** [comment]
PR #1606 
Works for ` this.setState({email: ''});` but not for `this.emailInput._root.clear();` because the clear function doesn't bubble up to the NB Input wrapper.

**Comment by SupriyaKalghatgi** [comment]
Floating Label does not reset position when input is cleared, this is desired behavior

**Comment by SupriyaKalghatgi** [comment]
Closing this issue

**Comment by SupriyaKalghatgi** [comment]
Released with [2.5.0](https://github.com/GeekyAnts/NativeBase/tree/v2.5.0)
